pyqtgraph/opengl/items/GLTextItem.py

```diff
@@ -0,0 +1,95 @@
+from OpenGL.GL import *
+import numpy as np
+from pyqtgraph.Qt import QtCore, QtGui
+from pyqtgraph.opengl.GLGraphicsItem import GLGraphicsItem
+import pyqtgraph.functions as fn
+
```

Comment:
To silence the static code analyzer, can you insert in here:

```python
__all__ = ['GLTextItem']
```

Reply:
I fixed in [e2aa1ef](https://github.com/pyqtgraph/pyqtgraph/pull/1776/commits/e2aa1ef733db223d0b28d68fa5f3568283250021).